entity-management-mongo/src/main/java/eu/europeana/entitymanagement/mongo/repository/EntityRecordRepository.java

```diff
@@ -69,7 +70,16 @@ public EntityRecord findByEntityId(String entityId) {
                 eq(ENTITY_ID, entityId))
                 .first();
     }
-
+    
+    public EntityRecordImpl checkForTheReference(String reference) {
```

Comment:
this is a dupplication of findMatchingEntitiesByCoreference which is already available in the develop branch, please move the implementation to the existing method

Reply:
Done.